### ISSUE TITLE
Document how consumers should identify browser telemetry

### ISSUE THREAD

**Issue by martinkuba** [open-telemetry/opentelemetry-specification#4145]
Related to https://github.com/open-telemetry/opentelemetry-specification/issues/2466

If consumers of telemetry wanted to distinguish browser telemetry from other types, they can use the value of `process.runtime.name` resource attribute (the value is always "browser" per [semantic conventions](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/process.md#javascript-runtimes)).

I would like to document this somewhere. Please advise where is the best place to do so.

**Comment by danielgblanco** [comment]
@open-telemetry/docs-maintainers do we have a specific section that you'd recommend for something like this? Personally, I think semconv "is" the place for it, but I can understand how it may be a little bit buried in detail for the end-user.

The best place I can think of would be something linking https://opentelemetry.io/docs/languages/js/getting-started/browser/ to the semconv docs.

**Comment by svrnm** [comment]
Yeah, this belongs into SemConv. Maybe here: https://opentelemetry.io/docs/specs/semconv/resource/ -- this page already has some notes on SDK defaults?
